content/sensu-go/6.2/release-notes.md

```diff
@@ -74,6 +75,42 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.2.0 release notes
+
+**December 17, 2020** &mdash; The latest release of Sensu Go, version 6.2.0, is now available for download.
+
+**ADD RELEASE SYNOPSIS HERE**
+
+See the [upgrade guide][1] to upgrade Sensu to version 6.2.0.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][193]) Added support for the `memberof` attribute for the [LDAP authentication provider][199].
+- ([Commercial feature][193]) Added the ability to exclude resource types when using sensuctl prune with the [--omit flag][200].
+- ([Commercial feature][193]) Added support for [round robin scheduling on PostgreSQL][201] instead of etcd.
+- ([Commercial feature][193]) Added support for OIDC authentication via [sensuctl configure][202].
+- The [/metrics API endpoint][196] now exposes build information as a Prometheus metric.
+- Added /health endpoint to agentd.
```

Comment:
```suggestion
- Added /health API endpoint to agent websocket transport.
```

Reply:
Good catch...don't know why I thought anyone would put together what the heck I was talking about there :D